sisl/viz/GUI/build/asset-manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Why do you have a `build` directory? This should be part of the installation process, no? Not the actual sources.

Reply:
This is one of the things that I did not know what to do. These are not binaries. It's just minified javascript and css for the browser to read it. So, they are equal for all users and they only need to be changed when there is an update interface.

They are generated from my code in order to reduce the size of the files to the minimum. To generate the build, you need [nodejs](https://nodejs.org/es/) and all the dependencies of my project. With all this, I believe it does not make sense to make the users generate the "build".

Where do you think I should keep it?

Reply:
I mean it works perfectly if it is separated from the sisl package. In fact this is what I was doing before bundling it all here so that people could test easily. It just adds a bit more complexity for the user to understand what to do to install it, but it can be done.

Reply:
Hmm... I need to think about this, I don't like shipping this in the sources...

Reply:
Maybe when the `GUI.launch()` method is used it could try to check if there are updates for the graphical interface and download all the code to store it in the same place that it is right now. 

I don't know how this can be done exactly but I bet it's possible. This would also keep the UI always updated even when the sisl package is not. But maybe at some point there are incompatibilities that make this approach not appropiate.

Reply:
I have something that might be good. The user would be able to access the GUI in two different ways:
- Use the interface **online**. The interface is **hosted somewhere**. Right now it's here just as an example: https://sisl-graphical-interface.web.app
- Use the interface **offline**. The interface is bundled with [electron](https://www.electronjs.org/) into a **native app for each OS**. Then people can download it from somewhere. For example: [App Image for Linux](https://www.dropbox.com/s/1l3gawwky9nx94v/ui-0.1.0.AppImage?dl=0)

In both cases the interface can interact either with the API that is running in localhost or with an API that is running anywhere else in the net if it's accessible. (The api can be run just by doing `python path/to/sisl/viz/GUI/api.py`)

In this way, we won't have the GUI files inside sisl but still it can be very easy for people to use it. What do you think? 

You wouldn't even need to have sisl installed to use the interface itself. Imagine the case that you are an experimental and some collaborator wants to share something with you live.  You would just need to go to https://sisl-graphical-interface.web.app, connect to their running API and then you could not only see the plots as they are building them, but also modify them using "their" sisl without any software requirement (not sisl, not even python).

Of course this would need a bit of security, but using API keys to verify the identity would not be that difficult.